### DISCUSSION TITLE
Reload url or just download from scratch correctly?

### DISCUSSION THREAD

**Discussion by pakiec22** [TumblThreeApp/TumblThree#281] [General]
**Expected Behaviour**

Lately I had older version of this app. I noticed, that it doesnt download all the content of the tumblr blog. I updated the app and tried again - with no positive result.
Then I deleted the url and added it once again. And only then it kinda worked and downloaded much more content, possiblly all the content.
So I needed to update the app + remove url and add the url again.
Only updating the app couldnt do the job.
So my question is, it should be done this way, that after updaing the app I am just making the already added url to be downloaded again I think?
Or maybe there should be some kind of reload/reset button or something? To reset the already added url so I can be processed from zero as the new url?

**Actual Behaviour**

It didnt do anything after the app update - not until I deleted the url and added it once again.

**Steps to Reproduce the Problem**

Old app with downloaded some blog where it doesnt download all the content and after the update it still doesnt - not without deleting and adding the url from zero.

**Specifications**

I dont really remember the previous version I had of this app sadly. But now I have the latest realease.

**Comment by thomas694** [comment]
Yes, you should always use the latest version, especially if you experience any problems or a sudden change in download capabilities.
But in this case, I think, the new version changed nothing related to your problem.

There are three modes that need to be distinguished.
In normal mode all new posts until the last successful download run are downloaded. So if two days since the last run have passed, only files of the last two days will be downloaded, nothing older is tried again.

If you know or believe that there are old posts that haven't been downloaded yet, you can use the second mode and tell the app to check all posts again by using the blog setting `Force rescan`.
In both modes no media files are downloaded again that already have been downloaded successfully before, even if you would delete the media file in windows explorer.

In the rare case that you really need to download a blog all over again, you can delete the blog's file index (a file like `blogname_files.tumblr` inside the `Index` folder) and maybe rename the blog's download folder. Then everything of this blog is downloaded again. In this way it keeps your blog's settings. Alternatively you can just remove the blog and add it again inside the app, as you did, but it uses the blog default settings again.

If I read your explanation correctly, you should have had success with the 'Force rescan' option.

**Comment by pakiec22** [comment]
1.Force rescan doesnt seem to be ideal.
I still find some tumblr blogs download more when deleted and added again.
There should be something more like reset blog or something button, so it really does start certain blog from the beginning. Like its added as new.

2.Plus is there blog import function? I can see export but nowhere import?

3.Plus it is pretty common for me to have crawls like this; [ ]
Where number of downloads is like 700, Downloaded files like 400 (these numbers are only examples) and Progress bar not at 100%. And it doesnt download anymore, it ends just there and another adding to queue or del/add doesnt help. It just ends that way like it finished completly. Plus adds date at last complete crawl. What is this about?
![173884609-7a20dbaf-6cd1-4d1b-8d4d-dcce6c05d93b](https://user-images.githubusercontent.com/71355143/173963811-e863283e-d0ee-4afb-9579-f836da125e95.jpg)

**Comment by thomas694** [comment]
We'll keep that in mind. But the question would be why it can download something if the blog is new and not when using rescan.

In the toolbar there is a button "Import blog list" or you can use the clipboard monitor to add a list.

This can happen for example if a blog contains plenty reblogs, either own or from other blogs. TumblThree does not download duplicates.
